packages/ui/libs/device/view/src/pages/RegisterPage/RegisterPage.effects.ts

```diff
@@ -27,6 +28,7 @@ export const useRegisterPageEffects = () => {
     useAllDeviceFuelTypes();
   const { allTypes: allDeviceTypes, isLoading: areDeviceTypesLoading } =
     useAllDeviceTypes();
+  const { myAccounts, isLoading: areMyAccountsLoading } = useApiMyAccounts();
```

Comment:
you should also check if there are cases when this endpoint is forbidden (e.g. user without organization, user with inactive organization, user without irec org) and add `enabled` prop. 
Also it should be disabled if `SINGLE_ACCOUNT_MODE` is false